app/assets/javascripts/surveys.js

```diff
@@ -443,5 +445,9 @@
   };
 
   root.GOVUK.userSurveys = userSurveys;
+
+  $(document).ready(function() {
+    GOVUK.userSurveys.init();
```

Comment:
Sure.  The other document ready functions don't but there's no reason we shouldn't pave the way.  Will do.